README.md

```diff
@@ -15,12 +15,17 @@ Things you may want to cover:
 ## Database creation
 
 ## Database initialization
+Once you have the proiject opened in VS Code. Install Postgres. Then Click the PostgresSQL Elephant icon, select the myapp_development dropdown menu, select the public drop down menu, right click users, hover over "Select..." and click "Run Select Top 1000". 
```

Comment:
When talking about how to access the data, include screenshots! Include the links to extensions you are recommending, make it as easy as possible for them